packages/langium/src/default-module.ts

```diff
@@ -30,35 +30,58 @@ import { DefaultNameProvider } from './references/naming';
 import { DefaultReferences } from './references/references';
 import { DefaultScopeComputation, DefaultScopeProvider } from './references/scope';
 import { DefaultJsonSerializer } from './serializer/json-serializer';
-import { LangiumDefaultServices, LangiumServices } from './services';
+import { DeepPartial, LangiumDefaultServices, LangiumDefaultSharedServices, LangiumGeneratedServices, LangiumGeneratedSharedServices, LangiumServices, LangiumSharedServices } from './services';
 import { DefaultDocumentValidator } from './validation/document-validator';
 import { ValidationRegistry } from './validation/validation-registry';
+import { ExtensionServiceRegistry, ServiceRegistry, SingleServiceRegistry } from './service-registry';
 
-export type DefaultModuleContext = {
+export interface DefaultModuleContext {
     connection?: Connection
+    serviceRegistry?: ServiceRegistry
 }
 
-export function createDefaultModule(context: DefaultModuleContext = {}): Module<LangiumServices, LangiumDefaultServices> {
+export function injectSingleService<T extends LangiumServices>(
```

Comment:
Can we provide a setup function that works well for an arbitrary number of languages? Otherwise the path from the single language (as we generate with `generator-langium`) to multiple languages is not clear for users.